plugin/eqdetect/source/dextool/plugin/eqdetect/runner.d

```diff
@@ -63,9 +63,23 @@ ExitStatusType runPlugin(string[] args) {
         auto s = SnippetFinder.generateKlee(visitor.function_params, source_path,
         mutant_path, visitor.function_name);
         writeToFile(s, baseName(m.path), m.kind, m.id, "_klee_");
+
+        import std.process;
+        import std.file: getcwd;
+        import std.format : format;
+
+        auto cwd = getcwd();
+
+        writeln(executeShell(format("docker run -it --name=klee_container -v %s:/home/klee/mounted klee/klee mounted/klee.sh", cwd)).output);
+        executeShell("docker rm klee_container");
+        executeShell("rm -rf eqdetect_generated_files/*");
+
         writeln("---------------------------------------");
```

Comment:
This solution is only temporary since running KLEE via docker is not the end goal. The problem with starting a script via a docker that compiles and runs KLEE is that it needs sudo privileges (as it is implemented now). This causes several issues, but will work for the moment when we are testing and continuously changing the implementation.

Reply:
Tip: if you arent interested in the captured output spawnShell may be used.
spawnShell(....).wait;